### PR TITLE
[40] added support to install externals from a directory and updated … [4-2-stable]

### PR DESCRIPTION
**Pull request by jassigill2000**

### CODE REVIEW

File: irods_consortium_continuous_integration_build_hook.py [hunk 1]
```diff
@@ -9,70 +10,53 @@
...
+
+def install_building_dependencies(externals_directory):
+    externals_list = ['irods-externals-cmake3.11.4-0',
+                      'irods-externals-avro1.9.0-0',
+                      'irods-externals-boost1.67.0-0',
+                      'irods-externals-clang-runtime6.0-0',
+                      'irods-externals-clang6.0-0',
+                      'irods-externals-cppzmq4.2.3-0',
+                      'irods-externals-json3.7.3-0',
+                      'irods-externals-jansson2.7-0',
```
Comment (trel) [inline irods_consortium_continuous_integration_build_hook.py:25]: we still need jansson?

Comment (jassigill2000) [reply irods_consortium_continuous_integration_build_hook.py:25]: Needs testing.

Comment (jassigill2000) [reply irods_consortium_continuous_integration_build_hook.py:25]: Just tested it we require jansson it is one of the dependencies

Comment (trel) [reply irods_consortium_continuous_integration_build_hook.py:25]: sure, it's listed/used, but the question is whether we need it.    the json3.7.3 library should do everything we need.

Comment (trel) [reply irods_consortium_continuous_integration_build_hook.py:25]: created #42

**Pull request merged**